cmd/serve.go

```diff
@@ -28,7 +32,21 @@ var (
 	accountRegex = regexp.MustCompile(`^\+[0-9]+$`)
 )
 
+func makeRandomClientId() string {
+	netInterfaces, err := net.Interfaces()
+
+	parts := []string{"signal-api-receiver"}
+
+	if err == nil {
```

Comment:
```suggestion
	parts := []string{"signal-api-receiver"}

	netInterfaces, err := net.Interfaces()
	if err == nil {
```

keep declaration next to error